src/components/AllJobLogsView/AllJobLogsView.js

```diff
@@ -51,48 +51,50 @@ const initialQuery = '?qindex=hrID&sort=-completedDate';
 const defaultSorting = '-completedDate';
 const defaultSortingSearch = '?sort=-completedDate';
 
-const buildJobsQuery = makeQueryBuilder(
-  `status=(${JOB_LOGS_STATUS_QUERY_VALUE})`,
-  query => `query=${query}`,
-  'sortby completedDate/sort.descending',
-  {
-    completedDate: buildDateTimeRangeQuery.bind(null, ['completedDate']),
-    startedDate: buildDateTimeRangeQuery.bind(null, ['startedDate']),
-    status: query => {
-      switch (true) {
-        case query === JOB_EXECUTION_STATUSES.FAIL:
-          return jobStatusFailString;
-        case Array.isArray(query):
-          return `status=(${query.map(v => (
-            v === JOB_EXECUTION_STATUSES.COMPLETED
-              ? JOB_LOGS_STATUS_QUERY_VALUE
-              : `"${v}"`)).join(' or ')})`;
-        default:
-          return `status=${query}`;
-      }
+const buildJobsQuery = (timezone) => {
+  return makeQueryBuilder(
+    `status=(${JOB_LOGS_STATUS_QUERY_VALUE})`,
+    query => `query=${query}`,
+    'sortby completedDate/sort.descending',
+    {
+      completedDate: buildDateTimeRangeQuery.bind(null, ['completedDate'], timezone),
+      startedDate: buildDateTimeRangeQuery.bind(null, ['startedDate'], timezone),
+      status: query => {
+        switch (true) {
+          case query === JOB_EXECUTION_STATUSES.FAIL:
+            return jobStatusFailString;
+          case Array.isArray(query):
+            return `status=(${query.map(v => (
+              v === JOB_EXECUTION_STATUSES.COMPLETED
+                ? JOB_LOGS_STATUS_QUERY_VALUE
+                : `"${v}"`)).join(' or ')})`;
+          default:
+            return `status=${query}`;
+        }
+      },
+      hrId: query => `hrid="${query}"`,
+      'runBy.userId': query => `runById="${query}"`,
     },
-    hrId: query => `hrid="${query}"`,
-    'runBy.userId': query => `runById="${query}"`,
-  },
-  {
-    hrId: 'hrid/number',
-    '-hrId': '-hrid/number',
-    totalRecords: 'total/number',
-    '-totalRecords': '-total/number',
-    errors: 'failed/number',
-    '-errors': '-failed/number',
-    exported: 'exported/number',
-    '-exported': '-exported/number',
-    updated: 'updatedDate',
-    '-updated': '-updatedDate',
-    jobProfileName: 'jobProfileName',
-    '-jobProfileName': '-jobProfileName',
-    runBy: 'runByFirstName runByLastName',
-    '-runBy': '-runByFirstName runByLastName',
-    startedDate: 'startedDate',
-    '-startedDate': '-startedDate',
-  }
-);
+    {
```

Comment:
Please revert this formatting, let's review our lint configuration first and allign with all other modules

Reply:
@vashjs , all rules are taken from `@folio/eslint-config-stripes`. We don't have any local rules for lint in` ui-data-export`